modules/updating-node-pools-for-hcp.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * updates/updating-hosted-control-planes.adoc
+// * hosted_control_planes/hcp-managing.adoc
+
+:_content-type: PROCEDURE
+[id="updating-node-pools-for-hcp_{context}"]
+= Updating node pools for hosted control planes
+
+On hosted control planes, you update your version of {product-title} by updating the node pools. The node pool version must not surpass the hosted control plane version.
+
+.Procedure
+
+* To start the process to update to a new version of {product-title}, change the `spec.release.image` value of the node pool.
```

Comment:
Is there meant to be a command output here?

Reply:
Good question, @kelbrown20. In the upstream docs that I used as my source, command output wasn't provided. 

@enxebre I'm wondering if it would be helpful to add an example command output for this procedure. Would you be able to provide one?

Reply:
You could
`oc -n NAMESPACE patch HC HCNAME --patch '{"spec":{"release":{"image": "foo"}}}' --type=merge`